### ISSUE TITLE
Build fails on fedora 27

### ISSUE THREAD

**Issue by nemobis** [the8472/fastar#1]
```
# rustc --version
rustc 1.25.0
# cargo --version
cargo 0.26.0
```

Failure:
```
ps/fastar-63f6471303058d4a.4xq48u46a1pwiqn7.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.4xuhwz5yy6q4gvgg.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.4ybye971cqflgun6.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.4yh8x2b62dcih00t.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.4ypvbwho0bu5tnww.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.51s1w397y42gpez1.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.533dixeydna25lub.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.5544xgl7axh7p8nz.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.555x2l3tz3it37mb.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.56dly8q07ws8ucdq.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.57k06xfugllsc526.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.57qy3vyd9bhiuaon.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.5beqqbexwb143v6a.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.5frs3mx5dzjbj7u6.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.63mmpc50mjcb0lb.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.7p53qlves2rwxx0.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.8xzrsc1ux72v29j.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.98g0d9x8aw3akpe.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.9elsx31vb4it187.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.ag4saemwliif5xv.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.c6lbtaiefvx3wya.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.e19kx9t7xhzajfl.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.e2z6uizknz84pv3.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.flypaog9cyk0inx.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.i9dq72iz2x71y1k.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.kt25z0521ngsjub.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.mz7vgmcf23rofcc.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.pqmjx25qpxkduz7.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.v6ozwtpojmqfurc.rcgu.o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.vpbpnff9uk7ha66.rcgu.o" "-o" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a" "/tmp/fastar/target/debug/deps/fastar-63f6471303058d4a.crate.allocator.rcgu.o" "-Wl,--gc-sections" "-pie" "-Wl,-z,relro,-z,now" "-nodefaultlibs" "-L" "/tmp/fastar/target/debug/deps" "-L" "/tmp/fastar/target/debug/build/miniz-sys-c5d1869982608274/out" "-L" "/usr/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "/tmp/fastar/target/debug/deps/libtar-53bf9b6cbbb734cb.rlib" "/tmp/fastar/target/debug/deps/libxattr-f70a5d5453b6213f.rlib" "/tmp/fastar/target/debug/deps/libfiletime-d79cb05b32ac68cf.rlib" "/tmp/fastar/target/debug/deps/libplatter_walk-0a62b3e76a3be715.rlib" "/tmp/fastar/target/debug/deps/libmnt-7543095ec032585e.rlib" "/tmp/fastar/target/debug/deps/libbtrfs2-11d7fd2131ebaf14.rlib" "/tmp/fastar/target/debug/deps/libuuid-36054f999b5e1dea.rlib" "/tmp/fastar/target/debug/deps/libminilzo-cd259c1d7375b3af.rlib" "/tmp/fastar/target/debug/deps/libminilzo_sys-3b7832d67aa3c240.rlib" "/tmp/fastar/target/debug/deps/libflate2-c26e84c544ce19ac.rlib" "/tmp/fastar/target/debug/deps/libminiz_sys-3271b853380c4c32.rlib" "/tmp/fastar/target/debug/deps/libcrc-fb0d012581b65f86.rlib" "/tmp/fastar/target/debug/deps/liblazy_static-baf548f49f326d68.rlib" "/tmp/fastar/target/debug/deps/libnix-55f950cda6f66623.rlib" "/tmp/fastar/target/debug/deps/libvoid-2f0b3cc6f9c94e2c.rlib" "/tmp/fastar/target/debug/deps/libcfg_if-931c28c291dea16c.rlib" "/tmp/fastar/target/debug/deps/libbitflags-265a94494a2ca4de.rlib" "/tmp/fastar/target/debug/deps/libreapfrog-1bb0c54233e854f3.rlib" "/tmp/fastar/target/debug/deps/libclap-cdff8e36bdf93dce.rlib" "/tmp/fastar/target/debug/deps/libatty-64b51839740f6207.rlib" "/tmp/fastar/target/debug/deps/libunicode_segmentation-1d3d7348db195ccb.rlib" "/tmp/fastar/target/debug/deps/libtextwrap-bb51618bca5e38b7.rlib" "/tmp/fastar/target/debug/deps/libterm_size-341c6f64a918f35a.rlib" "/tmp/fastar/target/debug/deps/liblibc-55853aa1f1add264.rlib" "/tmp/fastar/target/debug/deps/libvec_map-faccea34e70fb8e6.rlib" "/tmp/fastar/target/debug/deps/libbitflags-9ce73f074f5fde0d.rlib" "/tmp/fastar/target/debug/deps/libunicode_width-c1100c0fcf59c784.rlib" "/tmp/fastar/target/debug/deps/libansi_term-6483b18f15ace6b0.rlib" "/tmp/fastar/target/debug/deps/libstrsim-94021ae14ac23844.rlib" "/usr/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-4f0ab23dc3f85c9d.rlib" "/usr/lib/rustlib/x86_64-unknown-linux-gnu/lib/libpanic_unwind-d951fbfc30407bdb.rlib" "/usr/lib/rustlib/x86_64-unknown-linux-gnu/lib/libunwind-257c36ad3a592183.rlib" "/usr/lib/rustlib/x86_64-unknown-linux-gnu/lib/liballoc_system-842a7fb4fe6533bb.rlib" "/usr/lib/rustlib/x86_64-unknown-linux-gnu/lib/liblibc-f1c85bb9fa7064a2.rlib" "/usr/lib/rustlib/x86_64-unknown-linux-gnu/lib/liballoc-23e0db2ac2fa2290.rlib" "/usr/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd_unicode-a59888409676dd11.rlib" "/usr/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcore-bc69923e8ba9e8cb.rlib" "/usr/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-4a42596a1d140498.rlib" "-Wl,-Bdynamic" "-l" "lzo2" "-l" "z" "-l" "lzo2" "-l" "lzo2" "-l" "util" "-l" "dl" "-l" "rt" "-l" "pthread" "-l" "gcc_s" "-l" "c" "-l" "m" "-l" "rt" "-l" "pthread" "-l" "util" "-l" "util"
  = note: /usr/bin/ld: cannot find -llzo2
          /usr/bin/ld: cannot find -lz
          /usr/bin/ld: cannot find -llzo2
          /usr/bin/ld: cannot find -llzo2
          collect2: error: ld returned 1 exit status


error: aborting due to previous error

error: Could not compile `fastar`.

Caused by:
  process didn't exit successfully: `rustc --crate-name fastar src/main.rs --crate-type bin --emit=dep-info,link -C debuginfo=2 -C metadata=63f6471303058d4a -C extra-filename=-63f6471303058d4a --out-dir /tmp/fastar/target/debug/deps -C incremental=/tmp/fastar/target/debug/incremental -L dependency=/tmp/fastar/target/debug/deps --extern derive_error=/tmp/fastar/target/debug/deps/libderive_error-4a036342fff3375a.so --extern platter_walk=/tmp/fastar/target/debug/deps/libplatter_walk-0a62b3e76a3be715.rlib --extern tar=/tmp/fastar/target/debug/deps/libtar-53bf9b6cbbb734cb.rlib --extern reapfrog=/tmp/fastar/target/debug/deps/libreapfrog-1bb0c54233e854f3.rlib --extern nix=/tmp/fastar/target/debug/deps/libnix-55f950cda6f66623.rlib --extern clap=/tmp/fastar/target/debug/deps/libclap-cdff8e36bdf93dce.rlib -L native=/tmp/fastar/target/debug/build/miniz-sys-c5d1869982608274/out` (exit code: 101)
```

**Comment by nemobis** [comment]
All good after installing `lzo-devel`. Might be worth documenting the dependency.

**Comment by nemobis** [comment]
the8472, 14/04/2018 20:14:
> Closed #1 <https://github.com/the8472/fastar/issues/1> via b5d309e 
> <https://github.com/the8472/fastar/commit/b5d309e2de98c7f9d9cd311039e110f48302d715>.

Thanks! Very appreciated.
